com.freerdp.FreeRDP.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
This shouldn't be desktop app given it has to be launched from CLI.

Reply:
ok, changed to `conosle-application`